pkg/model/stack.go

```diff
@@ -839,3 +841,15 @@ func setEnvironmentFromFile(svc *Service, filename string) error {
 
 	return nil
 }
+
+func (s composeServices) toGraph() graph {
+	g := graph{}
+	for k, v := range s {
```

Comment:
WDYT about give more context about the type of the key and value from composeServices? would help when reading this lines. (svcName, svc or something like that)